src/control/server/storage/bdev/backend_json_test.go

```diff
@@ -165,6 +168,25 @@ func TestBackend_newSpdkConfig(t *testing.T) {
 				}...),
 			vosEnv: "AIO",
 		},
+		"multiple controllers; acceleration set to spdk; move and crc opts specified": {
+			class:        storage.ClassNvme,
+			devList:      []string{common.MockPCIAddr(1), common.MockPCIAddr(2)},
+			accelEngine:  "spdk",
+			accelOptCRC:  true,
+			accelOptMove: true,
+			expBdevCfgs:  multiCtrlrConfs(),
+			expDaosCfgs: []*DaosConfig{
+				{
+					Method: storage.ConfSetAccelProps,
+					Params: AccelPropsParams{
+						AccelEngine:  "spdk",
+						AccelOptMove: true,
+						AccelOptCRC:  true,
+						AccelOpts:    0b11, // verify bitmask calculated on Validate()
```

Comment:
This is coming out of the underlying C call, isn't it? Not urgent to fix, I don't think, but this is the kind of thing I would prefer be mocked out.